documentation/tutorials/buildlinux.rst

```diff
@@ -0,0 +1,58 @@
+=================================
+  How to build the Linux kernel
+=================================
+
+.. contents::
+   :depth: 1
+   :local:
+   :backlinks: none
+
+.. highlight:: console
+
+Get a Linux repository
+----------------------
+After successfully installing kw, it's time to build the Linux kernel using it.
+Let's start by cloning the Linux repository from Torvalds into our current
+working directory::
+
+  git clone https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git
+
+Then, change the current directory to the Linux repository::
+
+  cd linux
+
+Get .config
+-----------
+To build the kernel, a `.config` file is necessary. This file contains
+configurations about the kernel compilation process.
+To retrieve this file from your own computer, run, in Debian::
```

Comment:
Technically, all distros can support both strategies to get a config file. In this sense, I recommend not mentioning a specific distro here.